documentation/modules/rn-2.2.adoc

```diff
@@ -94,3 +94,37 @@ The error status message for a VM with no operating system on the *Migration pla
 .Network, storage, and VM referenced by name in the Plan CR are not displayed in the web console.
 
 If a Plan CR references storage, network, or VMs by name instead of by ID, the resources do not appear in the {project-short} web console. The migration plan cannot be edited or duplicated. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1986020[*BZ#1986020*])
+
+.Restart of the CatalogSource pod is required to upgrade MTV 2.1.0 to MTV 2.2.0.
+
+To restart the CatalogSource pod, do the following:
+
+.  In the *Subscriptions* tab of the {operator-name} in the {ocp-short} web console, take note of the name in the *CatalogSource* field.
```

Comment:
Explain where the tab is. 
UI elements use the preposition "on". See IBM Style Guide.

Reply:
On second thought -- why does the user need to find the pod name if they are getting the pods with `$ oc get pod ...`?

Reply:
To know which pod should be restarted. The pod name is in this format <the_catalogSource_ame-5charID>
For example: redhat-operators-mgrrm
**redhat-operators** is the catlogSource name used to install MTV 2.1